compiler/test/formatter_outputs/aliases.gr

```diff
@@ -18,18 +18,15 @@ type AReallyReallyReallyReallyReallyReallyReallyReallyReallyLongLineBreakingType
 type AReallyReallyReallyReallyReallyReallyReallyReallyReallyLongLineBreakingType<
   a,
   b
-> = String // Test comment
```

Comment:
Why does this disappear?

Reply:
Basically, the locations of the type parameters were just completely wrong before 😶

They're correct now, and the comments were only kept coincidentally before. @marcusroberts is already working on a fix for me!

Reply:
I think this is working now!